docs/earthly-config/earthly-config.md

```diff
@@ -58,6 +58,11 @@ earthly config global.cache_size_mb 20000
 
 Specifies the total size of the BuildKit cache, in MB. The BuildKit daemon uses this setting to configure automatic garbage collection of old cache. A value of 0 causes the size to be adaptive depending on how much space is available on your system. The default is 0.
 
+### cache_size_pct
```

Comment:
Could you open up a seperate PR against the `next` branch for the docs update?

Then when we do a release, we'll then merge `next` into `main`.

This is to prevent our docs from getting ahead of our currently released version of earthly.

Reply:
Makes perfect sense.  Will do that in a little bit.

Reply:
Done in #1737